src/desktop/components/artwork_filter_2/templates/index.jade

```diff
@@ -24,13 +24,13 @@
     #artwork-filter-right__results-container
       #artwork-filter-right__columns
         //- Rendered separately
-      .link-and-spinner
-        .artwork-spinner-container
-          .loading-spinner
+    .link-and-spinner
+      .artwork-spinner-container
+        .loading-spinner
```

Comment:
Noticed this in the faded-out-while-loading results and since we dropped the opacity down it looked kinda odd